app/grants/serializers.py

```diff
@@ -96,28 +96,6 @@ class Meta:
 
         fields = ('amount', 'asset', 'usd_value', 'timestamp', 'round')
```

Comment:
Consider: Now that we're including all chains, would it make sense to include the chain that the asset is associated with in the output?

Reply:
This is for pubic consumption! 
Do you see value adding it on? I figured we'd provide it if folks need it.
The data provided was what folks initially wanted